### ISSUE TITLE
"also forked to" appears although time threshold is not met

### ISSUE THREAD

**Issue by CennoxX** [musically-ut/lovely-forks#78]
I have "Only show message when last commit in base repo is older than?" set to 100 days, minimum stars to 2, but "also forked to" appears nevertheless.
![grafik](https://user-images.githubusercontent.com/6975881/135234721-5e05f0fa-6450-42cb-8203-d011d04c8975.png)
![grafik](https://user-images.githubusercontent.com/6975881/135234763-6d9655f1-2af8-487b-b0c2-8dcbc74734ff.png)

**Comment by musically-ut** [comment]
Indeed, I'm able to reproduce this. Initially, I'd thought that it could be because of caching, but something more sinister seems to be at play. 

I'll investigate it a bit more. Thanks for the report!
